### PR TITLE
Package-list shown info additions

### PR DESCRIPTION
**Pull request by elibarzilay**
Show in the package list indicators for per-package author-is-owner, and
owner approvals.  These are always shows, so it makes the information
available for maintainers to get a better picture of the PR and its
approvals.

### CODE REVIEW

**Comment by elibarzilay** [comment]
Not for merging ATM: this is a PR for just the top commit, stuff below that are on earlier PRs.

File: src/compute-pr-actions.ts [hunk 1]
```diff
@@ -180,23 +181,24 @@ function extendPrInfo(info: PrInfo): ExtendedPrInfo {
...
-        canBeMerged, approved, approverKind, requireMaintainer, blessable, failedCI, staleness,
+        canBeSelfMerged, pendingCriticalPackages, approved, approverKind, requireMaintainer, blessable, failedCI, staleness,
         packages, hasMultiplePackages, hasTests, newPackages, hasNewPackages,
         approvedReviews, changereqReviews, staleReviews, approvalFlags, hasChangereqs,
         reviewColumn, isAuthor
     };
 
     function getStaleness() {
-        return canBeMerged
+        return canBeSelfMerged
```
Comment (orta) [inline src/compute-pr-actions.ts:201]: Thanks!

File: src/compute-pr-actions.ts [hunk 2]
```diff
@@ -421,69 +431,79 @@ function createWelcomeComment(info: ExtendedPrInfo) {
...
+            display(`  - owner-approval: ${usersToString(approvers)}`);
         }
+        const displayOwners = (what: string, owners: string[]) => {
+            if (owners.length === 0) return;
+            display(`  - ${announceList(`${what} owner`, owners)}: ${usersToString(owners)}`);
+        };
+        displayOwners("added", p.addedOwners);
+        displayOwners("removed", p.deletedOwners);
+        if (!info.authorIsOwner && p.owners.length >= 4 && p.addedOwners.some(info.isAuthor)) addedSelfToManyOwners++;
+    }
```
Comment (orta) [inline src/compute-pr-actions.ts:483]: Cool work

**Review by orta — APPROVED** [review]
Cool, looks good.

**Pull request merged**